tools/install-powershell.sh-README.md

```diff
@@ -66,29 +66,3 @@ bash <(wget -O - https://raw.githubusercontent.com/PowerShell/PowerShell/master/
 ### Installation To do list
 
 * Detect and wait when package manager is busy/locked? - at least Ubuntu (CentOS does this internally)
-
-# install-powershell.ps1
```

Comment:
renaming the file make break links in the repo.  they may only be found when we merge to master.

Reply:
Full text search for `install-powershell-readme.md` finds no matches in the repo,